### DISCUSSION TITLE
Runner is not alive

### DISCUSSION THREAD

**Discussion by Jaspher16** [getgauge/taiko#2665] [General]
I am writing a straight forward test and it was working fine. But after certain executions, i am facing runner is not alive error message.

Could you please help me how to resolve it? I am not able to run further scripts

  Error Message: error reading from server: read tcp 127.0.0.1:51217->127.0.0.1:51216: wsarecv: An existing connection was forcibly closed by the remote host.
  Stacktrace: 
  
  Error Message: Runner is not Alive
  Stacktrace: 
  
Successfully generated html-report to => 

Failed to kill Runner: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:51216: connectex: No connection could be made because the target machine actively refused it."
Specifications:	0 executed	0 passed	0 failed	0 skipped
Scenarios:	0 executed	0 passed	0 failed	0 skipped

**Comment by marques-work** [comment]
Could you give some specifics about the host?

Are you running in a container, VM, or on bare metal?
Versions of Taiko, Chromium, and any other related software to this test?
Is it randomly failing, or always on the same types of tests?
Any system logs to corroborate?
